GOVERNANCE.md

```diff
@@ -64,10 +64,6 @@ A guide for Collaborators is maintained in
 ## CTC Membership
 
 CTC seats are not time-limited.  There is no fixed size of the CTC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
```

Comment:
> I feel like what's stated there is a given and so doesn't need to be stated explicitly. Of course we're looking to cover all important areas of expertise. Of course we want to be able to operate efficiently.

If it's not stated, it becomes assumptions. I don't have an immediate suggestion on how to make it better though, other than also covering "use-cases".